### DISCUSSION TITLE
How to add another row to the list table to the list.blade

### DISCUSSION THREAD

**Discussion by feniksdv** [Laravel-Backpack/community-forum#448] [Q&A (Help)] [answered]
Hello everyone. I use backpack 5 and laravel 10. I have a model that CRUD works with, and data from the model is output to list.blade.

And I also have an array with the same structure, I want to add it to the output of the same list.blade.

how do I do this?

**Comment by jcastroa87** [comment, marked as answer]
Hello @feniksdv,

Thank you for your question. It would be helpful if you could provide more details or even an image about the specific requirements you have. However, based on my understanding, it seems that you would like to modify the data displayed in a datatable.

To achieve this, you can override the "search" method on the "ListOperation" as described in our [documentation](https://backpackforlaravel.com/docs/5.x/crud-operation-list-entries#the-search-logic-1). This will allow you to customize the query that retrieves the data for the datatable. You can also apply any necessary filters, sorting, or pagination logic here.

If you would like to fully customize the list view, you can follow the instructions provided in our [documentation](https://backpackforlaravel.com/docs/5.x/crud-operation-list-entries#the-view-1). This will enable you to modify the HTML markup and styling of the datatable, as well as any other components on the page.

Please let us know if you require any further assistance.
